docs/sviluppo-mobile/README.md

```diff
@@ -0,0 +1,45 @@
+# Sviluppo Mobile
+
+## Che cos'è lo sviluppo mobile?
+
+Per "sviluppo mobile" si intende un insieme di processi e attività che coinvolgono l'analisi, la progettazione, la realizzazione e il rilascio di una o più applicazioni la cui esecuzione avverrà su dispositivi tascabili come smartphone e tablet.
+
+## Le fasi dello sviluppo mobile
```

Comment:
Si potrebbe aggiungere che mentre nello sviluppo web per utilizzare l'applicativo è necessario un browser, nello sviluppo mobile è necessario installare l'applicazione sul dispositivo (quindi menzionare le estensioni .ipa ed .apk).

Reply:
@fulgido interessante! Magari si potrebbe fare anche un breve accenno agli ISA

Reply:
Sarebbe interessante aggiungere oltre a quanto segnalato da @fulgido che anche la preparazione di un'ambiente di sviluppo non è così scontata, o forse poi diventa troppo tecnico e specifico? Obiettivamente credo che se si mette a sistema un "hello world" in C# o in Javascript contro uno in Flutter, ci sono delle differenze sostanziali nella preparazione dell'ambiente